packages/scaffold-config/src/frameworks.ts

```diff
@@ -181,6 +185,7 @@ export const WIZARD_FRAMEWORKS = [
     codeGenFramework: 'react',
     glob: '*.{js,jsx,tsx}',
     mountModule: 'cypress/react',
+    componentIndexHtml: componentIndexHtmlGenerator(),
```

Comment:
```suggestion
    componentIndexHtml: componentIndexHtmlGenerator('<div id="__next_css__DO_NOT_USE__"></div>')
```

We have the `react` and `next` templates swapped.

Reply:
Whoops. Fixed in upcoming commit.

Reply:
This was added to `nuxt` instead `next`!!

Reply:
Fix for realzies this time! ^^;;